ffi-cdecl/posix_decl.c

```diff
@@ -67,7 +67,7 @@ cdecl_func(usleep)
 cdecl_func(statvfs)
 cdecl_func(gettimeofday)
 cdecl_func(realpath)
-cdecl_func(basename)
+cdecl_func(basename) // NOTE: We'll want the GNU one (c.f., #4543)
```

Comment:
Might want to use the URL or otherwise make it clear that it's not that one over here. (Admittedly, it'll be a little while until we reach it. But still, #804 already!)